tests/conftest.py

```diff
@@ -411,6 +411,14 @@ def minicircle_grain_gaussian_filter(minicircle_grain_threshold_otsu: np.array)
     return minicircle_grain_threshold_otsu
 
 
+@pytest.fixture
+def minicircle_grain_z_threshold(minicircle_grain_gaussian_filter: np.array) -> Grains:
+    """Apply Z-scale thresholding."""
+    minicircle_grain_gaussian_filter.zrange = [0, 3]
```

Comment:
If you add the following after line 370 when you instantiate the `Grains()` class you shouldn't need this here.

```python
        zrange = grain_config["zrange"]     
```

Reply:
Ah perfect! I never saw that Grain() part!